src/odemis/driver/xt_client.py

```diff
@@ -1339,8 +1691,8 @@ def __init__(self, name, role, parent, **kwargs):
         self._updatePosition()
 
         # Refresh regularly the position
-        self._pos_poll = util.RepeatingTimer(5, self._refreshPosition, "Position polling")
-        self._pos_poll.start()
+        # self._pos_poll = util.RepeatingTimer(5, self._refreshPosition, "Position polling")
+        # self._pos_poll.start()
```

Comment:
@pieleric What should I do with these lines?

Reply:
Since we turned off event registration to fix the channel problems we had, I don't think listening to the FWDActualValueChanged event works. 

I will test your suggestions.

Reply:
I was reading the logs wrong. I saw there was 5 seconds between this call and the next, however that was just because there was nothing called in between. The actual call takes less than 0.5 seconds.

Regarding the events, Michal specifically discouraged the use of events because they are unreliable.